pages/AdListPage/[id]/index.js

```diff
@@ -40,24 +40,33 @@ export default function AdDetailsPage() {
           alt={`Examplephoto of ${ad.adTitle}`}
         />
       </ImageContainer>
-      <h3>{ad.adTitle}</h3>
+      <AdTitle>{ad.adTitle}</AdTitle>
       <Description>
         <p>Task description:</p>
         {ad.adDescription}
       </Description>
       <Attributes>
-        <p>category: {ad.category}</p>
+        <p>category:</p>
+        <CategoryItem> {ad.category}</CategoryItem>
         <p>Tags: </p>
         <TagsList>
           {ad.tags.map((tag, index) => {
-            return <li key={index}> {tag} </li>;
+            return <TagItem key={index}> {tag} </TagItem>;
           })}
         </TagsList>
         <p>Costs: {ad.adCosts}</p>
       </Attributes>
-      <StyledButton type="button" onClick={() => router.push("/")}>
-        go back
-      </StyledButton>
+      <ButtonContainer>
+        <StyledButton type="button" onClick={() => router.push("/")}>
+          go back
+        </StyledButton>
+        <StyledButton
+          type="button"
+          onClick={() => router.push(`/AdListPage/${id}/ContactPage`)}
+        >
+          Book now
+        </StyledButton>
```

Comment:
As mentioned before, these should be rather `Link` components instead of `buttons` as you are redirecting the user to another page.